lib/views/widgets/About/about_text.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+
+class AboutTextWidget extends StatefulWidget {
+  const AboutTextWidget({Key? key}) : super(key: key);
+
+  @override
+  _AboutTextWidgetState createState() => _AboutTextWidgetState();
+}
+
+class _AboutTextWidgetState extends State<AboutTextWidget> {
+  @override
+  Widget build(BuildContext context) {
+    double totalHeight = MediaQuery.of(context).size.height;
+    double totalWidth = MediaQuery.of(context).size.width;
+
+    return Container(
```

Comment:
it is better to put a singlechildscrollview .because of lots of text will be coming later